app/components/datetime-selector.js

```diff
@@ -0,0 +1,58 @@
+import Ember from "ember";
```

Comment:
A few code smells in this file, I note:

- the presence of `selectedDateTime`, `selectedTime ` AND `selectedDate`
- the presence of an observer to update data of a field (a property with a setter would probably work better here)
- a circular reference between a computed property and an observer

Not exactly sure what the goal of this piece of code is, but I have a feeling there are improvements that can be made.

But I'll approve the PR regardless, mostly because it's the last day and we need the features out.

Reply:
sure will work on it @patrixr and merge this PR for now.